content: add livementor case study (#1466)
nuxt/nuxtjs.org: content/en/blog/case-study-livementor.md

To make the transition progressive, we built micro Vue apps functioning on page levels, with a wrapper still rendered by Rails. But loading times were increasing, and we then knew we had to go a step further.

We found Nuxt at this time, via Twitter, and made a Proof of Concept project that went very well. Since then we're moving all our pages to Nuxt progressively.

### So you're making a progressive transition?

Yes, and we're now close to the end. We really wanted to reduce that transition phase, so we extracted features "as-is" to Nuxt. It's important to say that Nuxt made it really easy to make this progressive transition, it all fits together very simply.

Regarding deployments, Nuxt also helps a lot by abstracting the Node parts. We're using Heroku, and with a well configured Docker, it was seamless.

### Are you using dynamic or static rendering? Why?

We use dynamic rendering because our pages are not static and we have a lot of content updated constantly, so we use SSR to do this because it's not really a static site.

> We are focused like crazy on reactivity because all our pedagogical experience, all the learning experiences is based on the messaging app between the mentor and the students. We had trouble with our previous messaging system, latency and messages not showing up.

Reviewer: ```suggestion
We are focused like crazy on reactivity because all our pedagogical experience, all the learning experience is based on the messaging app between the mentor and the students. We had trouble with our previous messaging system, latency and messages not showing up.
```

The Firebase plus Nuxt combo made it much better.

![LiveMentor dashboard](case-study-livementor/mockup-m1-3.png)

### What is your favorite feature?

> The first feature that made us choose Nuxt was the project's architecture. Be it file-based routing or folders separation, it makes all fit in an intuitive way. Just dropping a file in your `pages` folder without having to configure a router is a must.

Reviewer: ```suggestion
The first feature that made us choose Nuxt was the project's architecture. Be it file-based routing or folder separation, it all fits in an intuitive way. Just dropping a file in your `pages` folder without having to configure a router is a must.
```

> For the moment, we use Vuex a lot, but we're thinking about moving a little bit from it by leveraging the Composition API.

Reviewer: ```suggestion
At the moment, we use Vuex a lot, but we're thinking about moving a little bit away from it by leveraging the Composition API.
```

> ### Do you have performances benchmarks before & after using Nuxt?

Reviewer: ```suggestion
### Do you have performance benchmarks before & after using Nuxt?
```

> On the migrated pages, we managed to cut by half our loading time. When we launched the new messaging, one of the first feedback from our users was "Wow this is really fast!".

Reviewer: ```suggestion
On the migrated pages, we managed to cut our loading time in half. When we launched the new messaging, one of the first comments we had from our users was "Wow this is really fast!"
```
